dataplane/standalone/saiserver/attrmgr/attrmgr.go

```diff
@@ -63,6 +68,27 @@ func (mgr *AttrMgr) set(id string, attr int32, val protoreflect.Value) {
 
 const protoNS = "lemming.dataplane.sai"
 
+// InvokeAndSave calls the RPC method and saves the attributes in the request.
```

Comment:
The description should indicate what it returns.

Reply:
done